server/src/main/java/io/spine/server/event/ERepository.java

```diff
@@ -74,14 +77,13 @@ public Event apply(@Nullable EEntity input) {
         checkNotNull(query);
 
         final EntityFilters filters = toEntityFilters(query);
-        final Iterable<EEntity> entities = find(filters, FieldMask.getDefaultInstance());
+        final Iterator<EEntity> entities = find(filters, FieldMask.getDefaultInstance());
         // A predicate on the Event message and EventContext fields.
         final Predicate<EEntity> detailedLookupFilter = createEntityFilter(query);
-        final Iterator<EEntity> entityIterator = FluentIterable.from(entities)
-                                                                   .filter(detailedLookupFilter)
-                                                                   .toSortedList(comparator())
-                                                                   .iterator();
-        final Iterator<Event> result = Iterators.transform(entityIterator, getEventFunc());
+        final Iterator<EEntity> filtered = filter(entities, detailedLookupFilter);
+        final List<EEntity> mutableList = newArrayList(filtered);
+        sort(mutableList, comparator());
+        final Iterator<Event> result = Iterators.transform(mutableList.iterator(), getEventFunc());
```

Comment:
Let's standardize on invoking `transform()`. Other parts of the code static import it. It would be better to have this line read:

```java
final Iterator<Event> result = transform(eventEntities, toEvent());
```
... which also assumes renaming `getEventFunc()` to `toEvent()`.